src/index.js

```diff
@@ -62,6 +63,9 @@ function App() {
         <Route path="/about">
           <About />
         </Route>
+        <Router path="/admin-dashboard">
```

Comment:
I already messaged about this, but thought I'd throw in here too :) 

Router --> Route, then we're good to merge!

Reply:
Changed it! I made changes to the branch that was already pushed to github. Good suggestion